packages/test/test-end-to-end-tests/src/test/gc/gcTombstone.spec.ts

```diff
@@ -578,25 +622,23 @@ describeNoCompat("GC tombstone tests", (getTestObjectProvider) => {
                 summarizingContainer,
                 summarizer,
             } = await summarizationWithUnreferencedDataStoreAfterTime(sweepTimeoutMs);
-
-            // Use the request pattern to get the testDataObject - this is unsafe and no one should do this in their
-            // production application - causes a sweep ready loaded error
-            const dataObject = await requestFluidObject<ITestDataObject>(summarizingContainer, unreferencedId);
+            await sendOpToUpdateSummaryTimestampToNow(summarizingContainer);
```

Comment:
I saw you didn't add this, why can you just not add this now? Does this have to do with the sweepready info loading immediately now?

Reply:
I mostly copy pasted one of the tests above and modified it. I must have missed this step.